### PR TITLE
Add View Extensions

### PR DESCRIPTION
**Pull request by brminnick**
This PR adds the docs for `ViewExtensions.cs`

### CODE REVIEW

File: docs/maui/TOC.yml [hunk 1]
```diff
@@ -183,3 +183,5 @@ items:
       href: markup/extensions/style.md
     - name: "VisualElement extensions"
       href: markup/extensions/visual-element-extensions.md
+    - name: "View extensions"
+      href: markup/extensions/view-extensions.md
```
Comment (bijington) [inline docs/maui/TOC.yml:187]: Minor alphabetical ordering fix

```suggestion
    - name: "View extensions"
      href: markup/extensions/view-extensions.md
    - name: "VisualElement extensions"
      href: markup/extensions/visual-element-extensions.md
```

File: docs/maui/markup/markup.md [hunk 2]
```diff
@@ -177,3 +177,4 @@ C# Markup extensions also allow developers to use an `enum` to define names for
 | [`Placeholder`](extensions/placeholder-extensions.md) | The `Placeholder` extensions provide a series of extension methods that support configuring `IPlaceholder` controls. |
 | [`Style`](extensions/style.md) | `Style<T>` provides a series of fluent extension methods that support configuring `Microsoft.Maui.Controls.Style`. |
 | [`VisualElement`](extensions/visual-element-extensions.md) | The `VisualElement` extensions provide a series of extension methods that support configuring the sizing, styling and behaviors of a `VisualElement`. |
+| [`View`](extensions/visual-element-extensions.md) | The `View` extensions provide a series of extension methods that support configuring the alignment of controls inheriting from `View`. |
```
Comment (bijington) [inline docs/maui/markup/markup.md:180]: Minor alphabetical ordering fix

```suggestion
| [`View`](extensions/visual-element-extensions.md) | The `View` extensions provide a series of extension methods that support configuring the alignment of controls inheriting from `View`. |
| [`VisualElement`](extensions/visual-element-extensions.md) | The `VisualElement` extensions provide a series of extension methods that support configuring the sizing, styling and behaviors of a `VisualElement`. |
```

**Review by bijington — APPROVED** [review]
Great thanks @brminnick. Just an unhelpful comment on the readability of some bits 😄

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Pull request merged**